manifests/params.pp

```diff
@@ -24,12 +24,11 @@
       $repo_gpg_key_source  = 'https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage'
 
       $server_package = $facts['os']['release']['major'] ? {
-        # RHEL 6 and 7 provide Gluster packages natively
-        /(6|7)/ => 'glusterfs-server',
+        /(6|7|8)/ => 'glusterfs-server',
         default => false
       }
       $client_package = $facts['os']['release']['major'] ? {
-        /(6|7)/ => 'glusterfs-fuse',
+        /(6|7|8)/ => 'glusterfs-fuse',
```

Comment:
we could also make this more readable my dropping the selector here and jus set `$client_package` to `glusterfs-fuse`. same for the server_package.